src/Installer/VendorInstaller.php

```diff
@@ -25,6 +25,7 @@ class VendorInstaller extends AbstractInstaller
     private Moodle $moodle;
     private MoodlePlugin $plugin;
     private Execute $execute;
+    private bool $noNvm;
```

Comment:
Silly detail, I'd keep the new attribute sort of "ordered", aka, after the `nodeVer` one. For your consideration.